CoreFoundation/Stream.subproj/CFStream.c

```diff
@@ -1761,7 +1761,9 @@ static CFRunLoopRef _legacyStreamRunLoop()
             pthread_attr_t attr;
             pthread_attr_init(&attr);
             pthread_attr_setdetachstate(&attr, PTHREAD_CREATE_DETACHED);
+#if !DEPLOYMENT_TARGET_LINUX
```

Comment:
Let's actually change this to DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED. We prefer the positive-sense style for deployment target macros in CF.